dependency/requirements.txt

```diff
@@ -1,5 +1,5 @@
 anyconfig~=0.10.0
-attrs~=21.3
+attrs>=21.3, <23.0  # The minimum version required for Airflow 2 is >=20.0 and Airflow 2.4.2 is >=22.1
```

Comment:
Maybe we shouldn't have an upper bound? https://www.attrs.org/en/22.1.0/changelog.html?highlight=upgrade#changelog They have a very strong backward-compatibility policy.

Reply:
Also, I'm not sure if we need to explain Airflow bounds here; the 21.3 is a Kedro-based lower bound, and we should just strive to be generally loose beyond that.

Reply:
I think it's ok to remove the upper bound since I don't remember `attrs` have ever created a problem for us. Though technically it wouldn't matter too much since it's a CalVer release.